ODBC/OAuthSampleApp.cpp

```diff
@@ -159,18 +160,20 @@ int main(int argc, char **argv)
     std::string clientId;
     std::string clientSecret;
     std::string tokenUrl;
+    std::string scope;
     static struct option long_options[] = {
         {"access-token", required_argument, 0, 'a'},
         {"refresh-token", required_argument, 0, 'b'},
         {"client-id", required_argument, 0, 'c'},
         {"client-secret", required_argument, 0, 'd'},
         {"token-url", required_argument, 0, 'e'},
+        {"scope", required_argument, 0, 'f'},
     };
     int c;
     while (1)
     {
         int option_index = 0;
-        c = getopt_long(argc, argv, "a:b:c:d:e:", long_options, &option_index);
+        c = getopt_long(argc, argv, "a:b:c:d:e:f", long_options, &option_index);
```

Comment:
I think it needs to have a colon after the "f" to denote it requires an argument.